.github/workflows/promote-beta-experimental-opt-in.yml

```diff
@@ -0,0 +1,37 @@
+# Promotes the current Nightly channel to Beta/Experimental opt-in on a schedule.
```

Comment:
Open to ideas on how to dedup these config files :)

Reply:
Some ideas -
1. using a reusable workflow - https://docs.github.com/en/actions/learn-github-actions/reusing-workflows
2. adding a matrix that keys on release channel. This would only work if jobs can be scheduled separately, I don't know whether it's supported

Reply:
Done. I hope. PTAL.